tsl/statements.go

```diff
@@ -318,3 +319,21 @@ func AttributesToString(attributes map[PrefixAttributes]InternalField) string {
 
 	return buffer.String()
 }
+
+// UnknownAttributesToString transform an UnkonwAttributes map into string
+func UnknownAttributesToString(attributes map[int]InternalField) string {
+	var buffer bytes.Buffer
+
+	buffer.WriteString("{")
+	prefix := ""
```

Comment:
instead of a buffer,
```go
attrs := make([]string, len(attributes))
i := 0
for k, v := range attributes {
    attrs[i] = fmt.Sprintf("%d:%s", k, v)
}
return fmt.Sprintf("{%s}", strings.Join(attrs, ", "))

```

Reply:
Updated